UtopiaCity/Models/FireSystem/ManagementSystemTransportAndEmployeess/Enums.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Models.FireSystem.ManagementSystemTransportAndEmployeess
+{
+    public class Enums
```

Comment:
You don't really need this class, also enums can be stored in simple cs file(create -> new -> code file), without class